data-import/guide-import-csv/guide-import-csv.adoc

```diff
@@ -10,282 +10,391 @@
+** if you have repeated ids across entities, provide the entity in parentheses, like `:ID(Order)`
+* `:START_ID`, `:END_ID` - relationship file columns referring to the node ids
+** for id-groups, use `:END_ID(Order)`
+* all other columns are treated as properties but skipped if empty or annotated with `:IGNORE`
+* type conversion is possible by suffixing the name, e.g. by `:INT`, `:BOOLEAN`, etc. (can see this in each of our files)
 
-The column names are used for property-names of your nodes and relationships, there is some extra markup for specific columns
+For more details on this header format and the tool, see the documentation in the link:{opsmanual}/tools/import/[Neo4j Manual^] and the accompanying link:{opsmanual}/tutorial/import-tool/[tutorial^].
 
-* `name:ID` - global id column by which the node is looked up for later reconnecting,
-** if property name is left off it will be not stored (temporary), this is what the `--id-type` refers to
-** if you have repeated ids across entities you have to provide the entity (id-group) in parentheses like `:ID(Order)`
-** if your ids are globally unique you can leave that off
-* `:LABEL` - label column for nodes, multiple labels can be separated by delimiter
-* `:START_ID`, `:END_ID` - relationship file columns referring to the node ids, for id-groups use `:END_ID(Order)`
-* `:TYPE` - relationship-type column
-* all other columns are treated as properties but skipped if empty or annotated with :IGNORE
-* type conversion is possible by suffixing the name, e.g. by `:INT`, `:BOOLEAN`, etc.
+[#data-load-quality]
+== CSV Data Quality
 
-For more details on this header format and the tool, see the extensive documentation in the link:{opsmanual}/tools/import/[Neo4j Manual] and the accompanying link:{opsmanual}/tutorial/import-tool/[tutorial].
+Real-world data is messy.
+Any time you work with data, you will see some values that need cleaned up or transformed before you move it to another system.
+We always recommend verifying data cleanliness yourself, even if another person has reviewed and approved it.
+Small syntax errors, format descriptions, consistency or correct quoting, and even differing assumptions on data requirements or standards can easily cause hours of cleanup down the road.
 
-And the resulting output would look like this:
+We will highlight some of the data quality issues easily missed when loading data from other systems into Neo4j and try to help avoid problems with data import and cleanup.
 
-[source, shell]
-----
-[INPUT------|ENCODER--------|WRITER: W:24.59 MB/s--------------------------]  4M
-Calculate dense nodes
-[INPU|CALCULATOR-----------------------------------------------------------]  3M
-Relationships
-[INPU|ENCODER-----------------------------------------------------------|WR]  4M
-Node first rel
+=== Common Pitfalls
+
+*Headers are inconsistent with data (missing, too many columns, different delimiter in header)*
```

Comment:
can you shorten these bullet points to make them more to the point/concise otherwise folks will skip over them 

but they are important

Reply:
Yep! Done.